### DISCUSSION TITLE
Zeichnen Symbol Drucken

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#206] [General]
Hallo,
ich bin gerade darauf aufmerksam gemacht worden, dass Symbole (Marker... unten das rote X) aus dem Zeichnen/Redlinig Tool beim Drucken "verschwinden" und ich habe es nachvollziehen können.
Ausdruck:
![image](https://github.com/user-attachments/assets/46eb1978-adfd-4e90-912b-cd4d241d8e95)
webGIS pro:
![image](https://github.com/user-attachments/assets/2f00591d-dbc8-48da-803d-f233586fa3a9)
lg

**Comment by gis-eni** [comment]
Konnte ich nachvollziehen. Der Bug wird mit Build >= 6.24.4001 behoben sein.
